exercises/practice/phone-number/.meta/example.8th

```diff
@@ -0,0 +1,17 @@
+: remove-punctuation \ s -- s
+  /[+()\-\s.]/ "" s:replace!
+;
+
+: digits \ s -- a
+  null s:/ ( >n null? if nip break ;then a:push ) [] a:reduce
```

Comment:
There must be some special behaviour for `[]`
```suggestion
  null s:/ ( >n null? if nip break ;then a:push ) a:new a:reduce
```

Reply:
Interesting. Thanks for figuring it out!